### ISSUE TITLE
Docker image

### ISSUE THREAD

**Issue by notdodo** [cyb3rfox/Aurora-Incident-Response#90]
Hi!

Do you plan to release a docker image or a docker compose for this project?

**Comment by gru3zi** [comment]
I would like this too

**Comment by digitalsleuth** [comment]
I have one just about ready, I'll keep you posted.

**Comment by cyb3rfox** [comment]
Cool, I plan to eventually also make that into a new version. May I get in touch with you regarding that when I start?

**Comment by digitalsleuth** [comment]
Absolutely, happy to help out! You can reach me offline through corey at digitalsleuth dot ca.

Once I get back to my comp I'll create a repo with the current Dockerfile and Compose file.

**Comment by akshaysth** [comment]
I'd love to assist with this lift too @cyb3rfox @digitalsleuth ! 😄

**Comment by digitalsleuth** [comment]
Okay, the initial framework (v1.0.0) is pushed to https://github.com/digitalsleuth/aurora-docker with the actual containers pushed as `latest` and `0.6.6`.

More to follow in the README, but in essence:
- Customize the docker-compose.yaml to your liking (mount points, SHM size, etc)
- `sudo docker-compose up -d` in the directory with the compose file
- `ssh -X aurora@0.0.0.0 -p 33` or whatever IP/PORT you choose in the compose file, password is `aurora`. The port is 33 to avoid collision with any possible SSH server in play on the current system, and the 0.0.0.0 is default. You can customize the network as you wish!
- Run `aurora` from the command prompt.

Exiting out of the docker leaves the container open for future connections / executions of aurora as necessary.

Again, this is just a preliminary, but it works for now :)
